mozregression/cli.py

```diff
@@ -386,13 +386,10 @@ def _convert_to_bisect_arg(self, value):
         try:
             value = parse_date(value)
         except DateFormatError:
-            try:
-                new_value = parse_date(date_of_release(value))
-                self.logger.info("Using date %s for release %s"
-                                 % (new_value, value))
-                value = new_value
-            except UnavailableRelease:
-                pass
+            new_value = parse_date(date_of_release(value))
```

Comment:
Ah I see, yes, we'll need to revise the patch then. I think it is probably best to just leave things as-is, but add some more logging information about mozregression is doing: whether it is bisecting by date, commit hash, or release number. Could you add that in?